src/librustc_typeck/diagnostics.rs

```diff
@@ -4146,8 +4102,6 @@ register_diagnostics! {
 //  E0068,
 //  E0085,
 //  E0086,
-    E0103, // @GuillaumeGomez: I was unable to get this error, try your best!
-    E0104,
```

Comment:
I don't think we usually remove these lints *outright*, but rather leave them commented out.. or at least that's what the surrounding lines seem to suggest. =) 

cc @rust-lang/docs -- what is the policy when an error code is no longer issued? (And what about if a "long-form" error exists, as in the case of E0102 above?) Is this documented somewhere? ([RFC 1567](https://github.com/rust-lang/rfcs/blob/master/text/1567-long-error-codes-explanation-normalization.md) doesn't seem to mention it.)